templates/cpan.conf.erb

```diff
@@ -15,14 +15,14 @@ $CPAN::Config = {
   'curl' => q[/usr/bin/curl],
   'ftp' => q[/usr/bin/ftp],
   'ftp_passive' => q[1],
-  'ftp_proxy' => q[],
+  'ftp_proxy' => q[<%=@ftp_proxy%>],
   'getcwd' => q[cwd],
   'gpg' => q[/usr/bin/gpg],
   'gzip' => q[/bin/gzip],
   'halt_on_failure' => q[0],
   'histfile' => q[/root/.cpan/histfile],
   'histsize' => q[100],
-  'http_proxy' => q[],
+  'http_proxy' => q[<%=@http_proxy%>],
```

Comment:
please check ``@ http_proxy `` is not undefined before adding this line in the config